nautobot/dcim/forms.py

```diff
@@ -190,7 +188,9 @@ def clean(self):
         elif mode == InterfaceModeChoices.MODE_TAGGED_ALL:
             self.cleaned_data["tagged_vlans"] = []
 
-        # Validate tagged VLANs; must be a global VLAN or in the same site
+        # Validate tagged VLANs; must be a global VLAN or in the same location
+        # TODO: after Location model replaced Site, which was not a hierarchical model, should we allow users to add a VLAN
+        # belongs to the parent Location or the child location of the parent device to the `tagged_vlan` field of the interface?
```

Comment:
Probably a future feature request. Good eye!

Reply:
@glennmatthews caught it!